02-peer-protocol.md

```diff
@@ -615,7 +615,6 @@ A sending node:
   - MAY send a `shutdown` before a `channel_ready`, i.e. before the funding transaction has reached `minimum_depth`.
   - if there are updates pending on the receiving node's commitment transaction:
     - MUST NOT send a `shutdown`.
-  - MUST NOT send multiple `shutdown` messages.
```

Comment:
Should't this instead be gated on awareness/negotiation of the new feature bit? Old nodes will still reject multiple shutdowns, so it should be feature gated.

Reply:
No, everyone always accepted multiple shutdown messages, because we *required* that they be retransmitted.  This requirement was always ignored!